homework-submission/react-hyf/server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "\"express, nodemon and concurrently for react-hyf\"",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "concurrently": "^4.1.1",
+    "express": "^4.17.1",
+    "nodemon": "^1.19.1"
```

Comment:
you have a `concurrently` and `nodemon` packages with no use, it will take more memory and disk space.

Reply:
Just delete from there will be enough?

Reply:
We install those ones in the class and remain there.

Reply:
that is what the homework was asking may be I did it in a wrong place.